test/recipes/80-test_ssl_old.t

```diff
@@ -445,6 +443,8 @@ sub testssl {
                 $flag = "-ssl3";
             } elsif ($protocol eq "TLSv1.2") {
                 $flag = "-tls1_2";
+            } elsif ($protocol eq "TLSv1.3") {
```

Comment:
Not sure what to comment...  I assume that when we don't add a specific protocol option, then anything goes, and if TLSv1.3 is enabled, then TLSv1.3 is part of the possibilities, and since the highest protocol version is chosen first (right?), ...